sass/_vars.scss

```diff
@@ -0,0 +1,4 @@
+$text_light: #FFFFFF;
+$text_dark: #6D6D6D;
+$header_font: "Helvetica Neue", Arial, sans-serif;
+$content_font: "Helvetica Neue", Arial, sans-serif;
```

Comment:
Here's how I would do this:
```scss
$text_light  : #FFF;
$text_dark   : #6D6D6D;
$header_font : Helvetica, "Helvetica Neue", Arial, sans-serif;
$content_font: $header_font;
```
The reason I use the variable here is so that you can use the already defined font, but if you need to separate them, you can simply do it from here by removing the `$header_font` variable and adding what you need.